src/components/transactionsV2/filterButton.js

```diff
@@ -172,7 +172,7 @@ class FilterButton extends React.Component {
               </div>
               <div className={styles.buttonContainer}>
                 <PrimaryButtonV2
-                  disabled={this.state.messageValidity}
+                  disabled={!!this.state.messageValidity}
```

Comment:
it is possible to use the disabled param without the double !!?

Reply:
This is to fix a warning, because `this.state.messageValidity` is a String and disabled expects a boolean value.

Reply:
oh ok good